plugin.xml

```diff
@@ -16,24 +16,51 @@ xmlns:android="http://schemas.android.com/apk/res/android"
         <engine name="cordova-ios" version=">=4.3.0"/>
     </engines>
 
+    <js-module name="FirebaseCrashPlugin" src="www/firebase-crash.js">
+      <clobbers target="FirebaseCrashPlugin" />
+    </js-module>
+
     <platform name="ios">
         <hook type="after_plugin_install" src="hooks/ios/after_plugin_install.js" />
         <hook type="before_plugin_uninstall" src="hooks/ios/before_plugin_uninstall.js" />
 
+        <config-file parent="/*" target="config.xml">
+          <feature name="FirebaseCrashPlugin">
+            <param name="ios-package" value="FirebaseCrashPlugin" />
+            <param name="onload" value="true" />
```

Comment:
To be honest, I'm not sure. Crashlytics itself should be active from the app start, logging crash data in case of a fatal app crash. So it may not be required. I'll test, if we really required unload to be true.

Reply:
okay, i think it's a good idea to have the plugin initialized at on app start. Imagine the use case when you're using a custom IonicErrorHandler which should log a non-fatal error. I suggest keeping unload to be true.